src/services/mongodb/index.ts

```diff
@@ -62,13 +63,7 @@ const path = `mongodb://${user}:${password}@${address}/${db}`
 
 export const setupMongoConnection = async (syncIndexes = false) => {
   try {
-    await mongoose.connect(path, {
-      autoIndex: false,
-      useNewUrlParser: true,
```

Comment:
Just curious, what's the context for us no longer needing these config params?

Reply:
The new mongoose version has new default settings AFAIR